discovery-infra/test_infra/assisted_service_api.py

```diff
@@ -248,9 +248,9 @@ def download_cluster_logs(self, cluster_id, output_file):
         with open(output_file, "wb") as _file:
             _file.write(response.data)
 
-    def get_events(self, cluster_id):
+    def get_events(self, cluster_id, host_id=''):
```

Comment:
I think that we should change it to waitForEvent because not all the events are raised synchronously - poll the event list every 5 seconds with a timeout

Reply:
Done, please review